packages/plugin/src/theia.d.ts

```diff
@@ -1649,6 +1655,21 @@ declare module '@theia/plugin' {
         lineNumbers?: TextEditorLineNumbersStyle;
     }
 
+    // Experimental API
+    // https://github.com/Microsoft/vscode/blob/1.30.2/src/vs/vscode.proposed.d.ts#L1015
+    export interface FileRenameEvent {
```

Comment:
@tolusha @benoitf How about keeping vscode.proposed feature into theia.proposed that we don't need to deprecate anything later unnecessary?